storage/create_bucket_upload_object/main.tf

```diff
@@ -0,0 +1,23 @@
+# Create new storage bucket in the US multi-region
+# with standard storage
+
+# [START storage_create_bucket_upload_object_tf]
+resource "google_storage_bucket" "static" {
+  project       = "<var>PROJECT_ID</var>"
+  name         = "<var>BUCKET_NAME</var>"
```

Comment:
Add a real bucket name using a random string.

Reply:
This code sample is going to be used in a Terraform quick start (https://cloud.google.com/storage/docs/terraform-create-bucket-upload-object#build_infrastructure), where we want to teach the user to create a bucket using their own bucket name. The sample isn't meant to be immediately copy/pasteable (I think it serves a different function than the sample on https://cloud.google.com/storage/docs/creating-buckets#create_a_new_bucket, let's say).

Reply:
@cbonnie Would you be able to use something like `example-bucket-for-proj-$data.google_project.org_id`

As mentioned in earlier comments, if bucket name is not unique it would fail during CICD testing.

Reply:
Using that line would remove a critical learning experience for users (learning how to create a bucket name using Terraform). We want to make sure we're providing a hands-on experience with users in the quickstart, instead of giving them something they can copy/paste without knowing how it works.

In regards to the testing: IMO, I think the test should be modified to accept var tags as arguments. Not being able to use var tags in Terraform code samples would mean that we can never use a Terraform code sample to teach users how to input values, and is inconsistent with the code sample experience we provide in other tools (see the gcloud instructions on how to create a bucket, which includes var tags: https://screenshot.googleplex.com/8d7x6ZephFSAWtA). TLDR: I don't think the test should limit how we educate users with Terraform code samples.

Reply:
Other doc sets are using var tags in Terraform code samples for quickstarts, too: https://cloud.google.com/vertex-ai/docs/tutorials/terraform/terraform-create-user-managed-notebooks-instance#build_infrastructure.

Not being able to put such code samples in GitHub would probably result in more difficult maintenance in the long run.

Reply:
Update: I had a quick pvt chat with @cbonnie

In `terraform-docs-samples` we discourage using vars in the interest of keep each region tag sections are as much possible (1) to be independent and (2) copy-paste-runnable. Only general exception is when we are referring to other data or resource type blocks. For example, a reference like `bucket = google_storage_bucket.static.id` is perfectly normal.

CC: @betsy-lichtenberg